index.html

```diff
@@ -701,7 +701,7 @@ <h2 class="red"> <a href="#">Latest deals > </a></h2>
               </section>
             </div>
     </main>
-    <footer class="footer">
+    <footer id="footer" class="footer">
```

Comment:
id names should be unique, also you should avoid using the same names as sematic tags for names of your classes and ids